install_config/topics/full_rhv_inventory.adoc

```diff
@@ -0,0 +1,112 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_rhv.adoc
+////
+
+= Example inventory for {product-title} on Red Hat Virtualization 
+
+The example inventory below assumes that the following items have been created:
+
+* A Load Balancer Instance
+* Three Master Instances (Optional: Part of a negative affinity group)
+** Extra disks attached: 15GiB for Docker, 30 GiB for local volume storage, and 25GiB for etcd
+* Three Infrastructure Instances (Optional: Part of a negative affinity group)
+** Extra disks attached: 15GiB for Docker, 30 GiB for local volume storage, and 25GiB for registry Gluster
+* One or more Application Instances (Optional: Part of a negative affinity group)
+** Extra disks attached: 15GiB for Docker, 30 GiB for local volume storage
+* DNS entries for the routers and for the OpenShift web console
+
+[source,yaml]
+----
+[OSEv3:children]
+nodes
+masters
+etcd
+glusterfs_registry
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
+openshift_deployment_type=openshift-enterprise
+openshift_release='3.10'
+openshift_master_cluster_method=native
+debug_level=2
+openshift_debug_level="{{ debug_level }}"
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_enable_service_catalog=False
+
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
```

Comment:
openshift_master_cluster_public_hostname could be just as same as "{{ load_balancer_hostname }}"
The line openshift_public_hostname=... here could be removed